setup.py

```diff
@@ -44,12 +44,15 @@
         'txtgen': ['regex', 'tensorflow'],
         'stanza': ['stanza==1.0.1'],
         'test': ['ddt', 'testfixtures'],
-        'example': ['termcolor==1.1.0'],
+        'example': ['termcolor==1.1.0', 'hypothesis==2.0.0'],
         'wikipedia': ['rdflib==4.2.2'],
         'ir': ['faiss-cpu>=1.6.1', 'elasticsearch==7.5.1'],
         'spacy': ['spacy==2.3.0'],
         'allennlp': ['allennlp==1.0.0', 'allennlp-models==1.0.0',
-                     'torch>=1.5.0,<1.6.0']
+                     'torch>=1.5.0,<1.6.0'],
+        'cliner': ['marisa-trie==0.7.4', 'scipy==1.5.0',
+                   'scikit-learn==0.23.1', 'repoze.lru==0.7',
+                   'tensorflow-gpu==1.12.0']
```

Comment:
So it looks like you have marisa-tree, repoze and similar packages here, you can definitely remove those from the commit.